packages/frontend/src/utils/project/rangeToLabel.ts

```diff
@@ -1,24 +1,40 @@
-import { assertUnreachable } from '@l2beat/shared-pure'
+import { rangeToDays } from '~/utils/range/rangeToDays'
+import type { ChartRange } from '../range/range'
 
 export function rangeToLabel(
```

Comment:
I am not quite sure if this usage is now correct because previously it was assumed that 123D means now-123D but now it's some random point in time -123D